cli/src/ui.rs

```diff
@@ -486,7 +486,7 @@ impl Ui {
         Ok(choice.starts_with(['y', 'Y']))
     }
 
-    pub fn prompt_password(&mut self, prompt: &str) -> io::Result<String> {
+    pub fn prompt_password(&self, prompt: &str) -> io::Result<String> {
```

Comment:
nit: can you update the other two `prompt_()` functions? They aren't used by git_util, but should be `&self` for consistency.